content/tutorials/manage-the-rippled-server/load-a-saved-ledger-in-stand-alone-mode.md

```diff
@@ -38,18 +38,20 @@ rippled stop --conf=/path/to/rippled.cfg
 
 ## 5. Start `rippled` in stand-alone mode.
 
-To load the most recent ledger version, you can use the [`-a`](https://wiki.ripple.com/Rippled#--standalone.2C_-a) and [`--load`](https://wiki.ripple.com/Rippled#--load) options when starting the server:
+To load the most recent ledger version, you can use the `-a` and `--load` options when starting the server:
 
 ```
 rippled -a --load --conf=/path/to/rippled.cfg
 ```
 
-To instead load a specific historical ledger, use the [`--load`](https://wiki.ripple.com/Rippled#--load) parameter along with the `--ledger` parameter, providing the ledger index or identifying hash of the ledger version to load:
+To instead load a specific historical ledger, use the `--load` parameter along with the `--ledger` parameter, providing the ledger index or identifying hash of the ledger version to load:
```

Comment:
I find starting the sentence with 'To instead...' a little cumbersome: How about:

> If you want to load a specific historical ledger, start the server with the `--load` and `--ledger` parameters, providing the ledger index or identifying hash of the ledger version to load:

Or just drop the 'instead':

> To load a specific historical ledger, start the server with the `--load` and `--ledger` parameters, providing the ledger index or identifying hash of the ledger version to load: